components/admin_console/group_settings/group_details/group_teams_and_channels_row.jsx

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import GlobeIcon from 'components/svg/globe_icon';
+import LockIcon from 'components/svg/lock_icon';
+
+export default class GroupTeamsAndChannelsRow extends React.PureComponent {
+    static propTypes = {
+        id: PropTypes.string.isRequired,
+        type: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+        implicit: PropTypes.bool,
```

Comment:
This comes from the concept of teams and channels in the group, an implicit team is a team that is implicitly included in the list because you have added channels. So, you haven't a explicit relation between the group and the team, but you have an implicit relation, because the a channel in that team is related to the group, so you will be included in the team to be included in the channel.